### ISSUE TITLE
TEM loss is not really decreasing

### ISSUE THREAD

**Issue by yt2639** [JJBOY/BMN-Boundary-Matching-Network#44]
Hey @JJBOY, thanks for your implementation and I found it extremely useful. I now have a problem during training, the PEM loss is decreasing normally, but TEM loss is not really decreasing, even it's increasing. Do you happen to have experience on how this may happen?

Thanks!

**Comment by JJBOY** [comment]
Thanks for your interests. TEM and&nbsp;PEM loss can not really reflect the final performance (e.g., AUC).&nbsp;Therefore, they are not of great reference value. They are printed just to see if training is going on properly.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年7月29日(星期四) 晚上9:00
收件人: ***@***.***&gt;; 
抄送: "Hello ***@***.***&gt;; ***@***.***&gt;; 
主题: [JJBOY/BMN-Boundary-Matching-Network] TEM loss is not really decreasing (#44)





 
Hey @JJBOY, thanks for your implementation and I found it extremely useful. I now have a problem during training, the PEM loss is decreasing normally, but TEM loss is not really decreasing, even it's increasing. Do you happen to have experience on how this may happen?
 
Thanks!

**Comment by wenba0** [comment]
i met this question too,loss didn't decrease on my own dataset,and even the official dataset.both of them the train loss is decreasing but val loss is not,how should i deal with it

**Comment by wenba0** [comment]
> so why the loss is not great reference value,and how cani make sure that my training is going on properly??some visualization?or some internal values?? THX
